resources/public/contracts/src/UsedByFactories.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity ^0.5.17;
  * @dev Provides modifiers to allow only offering factory contracts to execute method
  */
 
-import "Ownable.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
:+1:

Reply:
So you're saying I also need to remove the original `Ownable.sol`. You're right!

Reply:
Hehe :) I didn't notice you didn't remove it, but yeah :D